ebuild-maintenance/git/text.xml

```diff
@@ -0,0 +1,322 @@
+<?xml version="1.0"?>
+<guide self="ebuild-maintenance/git/">
+<chapter>
+<title>Git for Gentoo Developers</title>
+
+<body>
+<p>
+This guide covers git usage instructions and policies specific to Gentoo ebuild
+development. It assumes that the readers possess basic git knowledge.
+For a generic guide, please see the official
+<uri link="https://git-scm.com/book/">git book</uri>.
+</p>
+
+
+<section>
+<title>Preparing a development checkout</title>
+<body>
+
+<subsection>
+<title>Cloning the gentoo.git repository</title>
+<body>
+
+<p>
+In order to commit or submit ebuild changes, you should work on top
+of the development git repository. In order to have push access, you should use
```

Comment:
Avoid the repetitive use of `in order to`. If you assume basic git knowledge, you can take advantage of the lingo and use something like: `The main ebuild development happens on the official git repository. You can push your changes only through the SSH protocol. Therefore, clone the repository using:`

Reply:
Done.